user/moar/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "moar"
+pkgver = "1.30.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    f"-ldflags=-X main.versionString=v{pkgver}",
+]
+hostmakedepends = ["go"]
+pkgdesc = "Pager designed to just do the right thing without any configuration"
```

Comment:
i'd prefer a terser and less opinionated description (most people don't configure `less` either), so just `Terminal pager program` should be fine